packages/katello/katello/katello-service-bash_completion.sh

```diff
@@ -1,7 +1,7 @@
 # Opts for every command
 _katelloservice_help_opts="-h --help"
 _katello_action_opts="restart stop start status list enable disable"
-_katello_services="rh-mongodb34-mongod postgresql qpidd qdrouterd squid tomcat tomcat6 pulp_workers pulp_celerybeat pulp_resource_manager pulp_streamer foreman-proxy smart_proxy_dynflow_core httpd puppetserver foreman-tasks goferd"
+_katello_services="rh-mongodb34-mongod postgresql qpidd qdrouterd squid tomcat pulp_workers pulp_celerybeat pulp_resource_manager pulp_streamer foreman-proxy smart_proxy_dynflow_core httpd puppetserver foreman-tasks goferd"
```

Comment:
`foreman-tasks` should be `dynflowd`. It's not related to el6 so I'd be fine with doing it later. It probably also works because it's aliased.